### PR TITLE
Strip template info, make app an app, and updates

### PR DESCRIPTION
**Pull request by archae0pteryx**
#### Take this baby from template to big boy/girl app

- set the app name (duh)
- update docs / remove template specific info
- removed template identifiers as they're just guff now
- I removed a ridiculous package `use-react`... we were using the `useToggle` hook and well.. it's literally just a setState so i removed it. You can look at the hook [here](https://github.com/streamich/react-use/blob/master/docs/useToggle.md)
- there was a bug with the playwright script being locked to the template name - fixed.
- i updated and locked our CI ubuntu / node versions. Previously we had an issue with the CI ubuntu image upgrading a package that borked us. We updated that particular action at the time but not the others so i did it for all
- I upgraded and locked our node versions to 18. I seem to recall a convo @craineum we had about why not to do this and i think that was pre redwood v3 or something? I know it's supported now so i threw it in. Can def remove though

### CODE REVIEW

File: .github/workflows/api.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   api_test:
     runs-on: ubuntu-20.04
-    container: node:16
+    container: node:18
```
Comment (iameddieyayaya) [inline .github/workflows/api.yml:14]: does RW work with Node 18?

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-v16
+v18
```
Comment (iameddieyayaya) [inline .nvmrc:1]: does RW work w/ node 18?

**Review by iameddieyayaya — APPROVED** [review]
LGTM 🚀

File: .nvmrc (hunk 2 above)
Comment (archae0pteryx) [reply .nvmrc:1]: @iameddieyayaya and @craineum If i remember correctly, redwood generates the engines config in package json to support >=16... I just looked and you're right it isn't supported. I will knock it back down. That's unfortunate. =( I swear i saw that somewhere but I shoulda looked. my bad.

File: web/src/pages/Admin/User/UsersPage/UsersPage.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
+import { useState } from 'react'
+
 import { MetaTags } from '@redwoodjs/web'
 
 import UsersCell from 'src/components/Admin/User/UsersCell'
-import { useToggle } from 'src/lib/hooks'
 
 const UsersPage = () => {
-  const [showInactive, toggleShowInactive] = useToggle(false)
+  const [showInactive, toggleShowInactive] = useState(false)
```
[the remark this answers is not in the document]
Comment (archae0pteryx) [reply web/src/pages/Admin/User/UsersPage/UsersPage.tsx:8]: I've got a list. =)

**Comment by archae0pteryx** [comment]
Now i see you've merged it with the node 18 change. Cool. Wont change back

**Pull request merged**